tasks/main.yml

```diff
@@ -292,7 +278,101 @@
       debug:
         var: __mssql_sqlcmd_input_file.stdout_lines
 
-    - name: Remove the {{ __mssql_sql_tempfile.path }} file
+    - name: Remove the tempfile
       file:
         path: "{{ __mssql_sql_tempfile.path }}"
         state: absent
+
+- name: Configure the sqlagent setting
+  include_tasks: mssql_conf_setting.yml
+  vars:
+    __mssql_conf_setting: "sqlagent enabled"
+    __mssql_conf_setting_value: "{{ mssql_enable_sql_agent }}"
+  when: mssql_enable_sql_agent is not none
+
+- name: Install the {{ __mssql_server_fts_packages }} package
+  package:
+    name: "{{ __mssql_server_fts_packages }}"
+    state: "{{ 'present' if mssql_install_fts else 'absent' }}"
+  when: mssql_install_fts
```

Comment:
You can't use a "bareword" here - use `when: mssql_install_fts | bool`

Reply:
Ok, will do

Reply:
Changed this to `is not none` as per Noriko's suggestion to make this work for both install and uninstall

Reply:
> Changed this to `is not none` as per Noriko's suggestion to make this work for both install and uninstall

Thank you, @spetrosi! How about adding `| bool` to the above line to ensure the value of mssql_install_fts is a valid boolean like this?
```
state: "{{ 'present' if mssql_install_fts | bool else 'absent' }}"
```

Reply:
Users must provide a boolean value with this `mssql_install_fts` variable, it is written in README. And I want to force users to do this because tasks depend on this. Now if a user provides e.g. a string "no", instead of boolean, the task will fail with a descriptive error message forcing users to use booleans:
```
fatal: [127.0.0.3]: FAILED! => {"changed": true, "cmd": ["/opt/mssql/bin/mssql-conf", "set", "sqlagent", "enabled", "no"], "delta": "0:00:00.058723", "end": "2021-04-19 04:38:53.632981", "failed_when_result": true, "rc": 0, "start": "2021-04-19 04:38:53.574258", "stderr": "", "stderr_lines": [], "stdout": "Validation error on setting 'sqlagent.enabled'\nenabled value must be '1' or '0'", "stdout_lines": ["Validation error on setting 'sqlagent.enabled'", "enabled value must be '1' or '0'"]}
```
If I add | bool there, the task will just do `absent` if a user provides non-boolean.